package.json

```diff
@@ -22,11 +21,11 @@
     "redis:start": "docker run -p 6379:6379 --name redis-mwoffliner -d redis",
     "redis:kill": "docker rm -f redis-mwoffliner || :",
     "redis": "npm run redis:kill && npm run redis:start",
-    "postinstall": "npm run build"
+    "postinstall": "npm run build",
+    "debug": "TS_NODE_FILES=true node --inspect -r ts-node/register src/cli.ts --mwUrl=https://bm.wikipedia.org --adminEmail=admin@kiwix.com --verbose --redis=redis://127.0.0.1:6379 --format=nozim --articleList=../articleList"
```

Comment:
This combination of `--articleList` and `bm.wikipedia.org` doesn't work (because the articles are for `en.wikipedia.org`).
Could you tweak that please?